scripts/collect-windows-logs.ps1

```diff
@@ -36,12 +38,17 @@ Get-CimInstance win32_pagefileusage | Format-List * | Out-File -Append "$ENV:TEM
 Get-CimInstance win32_computersystem | Format-List AutomaticManagedPagefile | Out-File -Append "$ENV:TEMP\\$($timeStamp)_pagefile.txt"
 $paths += "$ENV:TEMP\\$($timeStamp)_pagefile.txt"
 mkdir 'c:\k\debug' -ErrorAction Ignore | Out-Null
-Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/windows/debug/collectlogs.ps1 -OutFile 'c:\k\debug\collectlogs.ps1'
+
+Write-Host "Collecting networking related logs"
+if (-not (Test-Path 'c:\k\debug\collectlogs.ps1')) {
+  Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/windows/debug/collectlogs.ps1 -OutFile 'c:\k\debug\collectlogs.ps1'

```

Comment:
Could we add some logs to measure how long it takes to download the file?

Reply:
This script is isn't called during CSE execution / provisioning.
Do you just want the time written to stdout/write-host?

Reply:
I see. I mixed them up. Please ignore it.